src/main.less

```diff
@@ -1,5 +1,9 @@
+@import url('./styles/roboto-font.less');
+@import url('./styles/dancing-script-font.less');
 @import url('@educandu/educandu/styles/main.less');
 
+@global-font-family: 'Roboto', sans-serif;
```

Comment:
I've set this as was for now (as 'Roboto' though set through educandu before), because the font change is part of the "make OMA blue" ticket.

Reply:
OK